doc/user-guide.adoc

```diff
@@ -68,6 +68,29 @@ spec:
   replicas:2
 ----
 
+[[resources]]
+## Specify the Resource requirements for the container
+
+The `resources` spec is defined in the link:../apis.adoc#Resources[Resources API Documentation].
```

Comment:
I am getting 404 as this is resolving to 
https://github.com/wildfly/wildfly-operator/blob/master/apis.adoc#Resources

Should be 
https://github.com/wildfly/wildfly-operator/blob/master/doc/apis.adoc#resources

Reply:
@jmesnil